sdk/ml/azure-ai-ml/azure/ai/ml/operations/_code_operations.py

```diff
@@ -176,7 +176,7 @@ def create_or_update(self, code: Code) -> Code:
                         target=ErrorTarget.CODE,
                         no_personal_data_message=CHANGED_ASSET_PATH_MSG_NO_PERSONAL_DATA,
                         error_category=ErrorCategory.USER_ERROR,
-                    )
+                    ) from ex
```

Comment:
Same for this; 'from e' or 'from ex'?

Reply:
No for same reason as other comment

https://github.com/Azure/azure-sdk-for-python/blob/e0a67a777ce69f1df5b0f2a8a5cbc8b7ef59acc6/sdk/ml/azure-ai-ml/azure/ai/ml/operations/_code_operations.py#L168-L180